pushapkscript/docker.d/worker.yml

```diff
@@ -72,10 +72,17 @@ products:
               package_names: [ "org.mozilla.firefox_beta" ]
               certificate_alias: 'fennec-beta'
               google:
-                # TODO Change to "production" once the Fenix Beta migration is done
-                default_track: 'alpha'
+                default_track: 'production'
```

Comment:
@JohanLorenzo I reckon so far we've been publishing to `alpha` track and RelMan/Mobile folks have promoted to `production`. If this goes to production, we're performing this step automatically. Are they expecting this, should we heads-up them? I'm wondering what the cadence for Fenix beta is these days.

Unrelatedly, kudos for all these comments left in code, I feel much safer reading some of the TODOs knowing I've reviewed some of these things in the past.

Reply:
Yeah, I wanted to ask on Friday's call, but I forgot. I'll get the approval first!

Reply:
We want to keep the alpha track for now. Full conversation at https://mozilla.slack.com/archives/CKM7DLL67/p1591024730316900